tests/test_pagination.py

```diff
@@ -217,15 +216,18 @@ def sqla_14_all_users_with_notes_count(paginate: Paginate = Depends()):
         )
         return paginate(query, scalars=False)
 
-    def query_count(session: Session, query: Select) -> int:
-        return session.execute(select(func.count()).select_from(user_cls)).scalar()
+    def count_user_notes(user_id: int, session: Session = Depends()) -> int:
+        return session.execute(
+            select(func.count(note_cls.id)).where(note_cls.user_id == user_id)
+        ).scalar()
 
-    CustomPaginate: PaginateSignature = Pagination(query_count=query_count)
+    CustomPaginate: PaginateSignature = Pagination(query_count=count_user_notes)
```

Comment:
can we type the return type of `Pagination` so that `PaginateSignature` is inferred?